packages/nodejs/src/tracer.ts

```diff
@@ -64,6 +70,30 @@ export class BaseTracer implements Tracer {
     return this.#scopeManager.active() || new NoopSpan()
   }
 
+  /*
+   * Returns the root Span.
+   *
+   * If ther is no root Span available, a NoopSpan is returned.
+   */
+  public rootSpan(): NodeSpan {
+    return this.#scopeManager.root() || new NoopSpan()
+  }
+
+  /*
+   * Adds the given error to the root Span.
+   *
+   * If there is no root Span available to add the error, `undefined` is returned.
+   */
+  public addError(error: Error): NodeSpan | undefined {
+    const activeRootSpan = this.rootSpan()
+
+    if (activeRootSpan instanceof NoopSpan) return
```

Comment:
Would calling any function on the noopspan not also prevent any action from being taken? I think we can remove this.